conjure-java-undertow-runtime/src/main/java/com/palantir/conjure/java/undertow/runtime/WorkerStartTimeHandler.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * (c) Copyright 2019 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.conjure.java.undertow.runtime;
+
+import io.undertow.server.HttpHandler;
+import io.undertow.server.HttpServerExchange;
+import io.undertow.util.AttachmentKey;
+
+public final class WorkerStartTimeHandler implements HttpHandler {
+
+    public static final AttachmentKey<Long> WORKER_START_TIME_KEY = AttachmentKey.create(Long.class);
+
+    private final HttpHandler next;
+
+    WorkerStartTimeHandler(HttpHandler next) {
+        this.next = next;
+    }
+
+    @Override
+    public void handleRequest(HttpServerExchange exchange) throws Exception {
+        exchange.putAttachment(WORKER_START_TIME_KEY, System.currentTimeMillis());
```

Comment:
The exchange already provides the start time. Since we're not consuming this value, I think this functionality is a better fit for individual webservers which produce full metric data. We already have a MetricsHandler in WC which duplicates this functionality.